cmd/mnemonic/mcp.go

```diff
@@ -48,8 +54,14 @@ Example:
 		mcp.RegisterListTagsTool(s, db)
 		mcp.RegisterSearchEntriesTool(s, db)
 
+		// Show which database is being used
```

Comment:
Let's get rid of these kind of low-signal comments in our code please. Each comment should explain why the code it is referring to is written that way. We should not have comments simple explaining what code does. We can understand that by reading the code. (Exceptions can be made only for very complicated things.)